extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -39,3 +37,9 @@ tags: {  # <amp-story-player>
     supported_layouts: RESPONSIVE
   }
 }
+descendant_tag_list {
+  name: "amp-story-player-allowed-descendants"
+  tag: "A"
+  tag: "SPAN"
```

Comment:
Is this one needed as well?

Reply:
Yep, as we recommend publishers to include `<span>` inside their `<a>` tags to provide a title/description of their stories.

Reply:
Do you have an example of that (the `<span>` usage)? https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md doesn't mention this

Reply:
Hmm I thought we wrote it as an example it in the [article here](https://amp.dev/documentation/guides-and-tutorials/integrate/embed-stories/) but doesn't look like it. Going to follow up on this.

I guess it's not a big deal if you don't use them, since they will be hidden anyways when the player loads. The only usage I can think of is setting styles for `<span>` before the JS loads, if that's important.